content/applications/hr/time_off.rst

```diff
@@ -392,65 +432,79 @@ information:
   allocation, enter it in this field at the bottom of the form.
 
+.. image:: time_off/new-allocation.png
+   :align: center
+   :alt: A new allocation form with all the fields filled out for the annual two week vacation
+         granted to all employees.
+
 .. _time_off/request-time-off:
 
 Request time off
 ================
 
 Once an employee has been allocated time off, a request to use the time off can be submitted. Time
 off can be requested in one of two ways, either from the :ref:`dashboard <time_off/dashboard>` or
-from the :guilabel:`My Time Off` view.
+from the :ref:`My Time Off <time_off/my-time-off>` view.
 
-To access the dashboard, navigate to :menuselection:`Time Off app --> My Time Off --> Dashboard`.
-This is also the default view for the *Time Off* application.
+To access the dashboard, navigate to :menuselection:`Time Off app --> My Time --> Dashboard`. This
+is also the default view for the *Time Off* application.
 
-To access :guilabel:`My Time Off`, navigate to :menuselection:`Time Off app --> My Time Off --> My
-Time Off`. This presents a list view of all the time off requests for the employee.
+To access :guilabel:`My Time Off`, navigate to :menuselection:`Time Off app --> My Time --> My Time
+Off`. This presents a list view of all the time off requests for the employee.
 
-To create a new request for time off, click either the :guilabel:`New Time Off` button on the main
-*Time Off* dashboard, or the :guilabel:`Create` button in the :guilabel:`My Time Off` list view.
-Both buttons open a new time off request form.
+To create a new request for time off, click the :guilabel:`New` button on either the main *Time Off*
+:guilabel:`Dashboard` or the :guilabel:`My Time Off` list view.
 
-Enter the following information on the form:
+.. note::
+   Both :guilabel:`New` buttons open a :guilabel:`New Time Off` request form, but when requested
+   from the :guilabel:`Dashboard`, the form appears in a pop-up window. When requested from the
+   :guilabel:`My Time Off` list view, the screen navigates to a new time off request page instead.
+
+Enter the following information on the :guilabel:`New Time Off` request form:
 
 - :guilabel:`Time Off Type`: select the type of time off being requested from the drop-down menu.
-- :guilabel:`Dates`: enter the dates that the time off will fall under. There are two fields to
-  populate, the :guilabel:`From` and :guilabel:`To` fields. Click on either the :guilabel:`From` or
-  :guilabel:`To` field, and a calendar pop-up appears.
+- :guilabel:`Dates`: enter the dates that the time off falls under. There are two fields to
+  populate, the start and end dates. Click on either date field and a calendar pop-up appears.
 
   Click on the start date, then click on the end date. The selected start and end dates appear in
-  deep purple, and the dates between them appear in pale purple (if applicable).
+  deep teal, and the dates between them appear in pale teal (if applicable).
```

Comment:
I am such a color-focused person, the idea to NOT mention the color didn't occur to me! =D  I agree- I think I was focused on the color because it changed from the previous version to this one. But I like your idea!